bw2data/revisions.py

```diff
@@ -1,5 +1,5 @@
 import json
-from typing import Any, Optional, Sequence, Union
+from typing import Any, Iterator, Optional, Sequence, Union
```

Comment:
```suggestion
from typing import Any, Optional, Sequence, Union
```

See below on the `return`